SECURITY.md

```diff
@@ -0,0 +1,48 @@
+# Security Policy
```

Comment:
@rutger-h Could you put this file in the /development git branch?

Right now its in the /main branch. All changes here are overriden by /development in the next release

Reply:
Done.

Reply:
Something went wrong here. 536 file have changed